lib/src/main/java/org/team100/lib/motion/drivetrain/manual/DriveWithNoteRotation.java

```diff
@@ -0,0 +1,54 @@
+package org.team100.lib.motion.drivetrain.manual;
+
+import org.team100.lib.localization.NotePosition24ArrayListener;
+import org.team100.lib.motion.drivetrain.kinodynamics.SwerveKinodynamics;
+import org.team100.lib.telemetry.Telemetry;
+import org.team100.lib.telemetry.Telemetry.Level;
+import org.team100.lib.util.DriveUtil;
+import org.team100.lib.util.Names;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Twist2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+/**
+ * Transform manual input into ChassisSpeeds.
+ * 
+ * The twist components, x, y, and theta, are mapped directly to the
+ * corresponding ChassisSpeeds components (and scaled).
+ */
+public class DriveWithNoteRotation {
+    private final PIDController m_pidController;
+    private final Telemetry t = Telemetry.get();
+    private final SwerveKinodynamics m_swerveKinodynamics;
+    private final String m_name;
+    private final NotePosition24ArrayListener notePosition;
+    public DriveWithNoteRotation(String parent, SwerveKinodynamics swerveKinodynamics, PIDController pidController, NotePosition24ArrayListener arrayListener) {
+        notePosition = arrayListener; 
```

Comment:
the name in the constructor argument list should be the same as the field name, except for the "m_"